frontend/www/sass/base/_variables.scss

```
@@ -127,12 +168,19 @@
   --notifications-clarifications-drawer-li-background-color--active: #678dd7;
   --notifications-clarifications-drawer-li-font-color--active: #ffffff;
   --notifications-notification-date-font-color: #666666;
+  --notifications-notification-link-background-color--hover: rgba(
+    0,
+    0,
+    0,
+    0.05
+  );
 
   // QUALITY NOMINATIONS
   --quality-nomination-popup-border-color: #cccccc;
   --quality-nomination-popup-background-color: #000000;
   --quality-nomination-tag-select-border-color: #cccccc;
   --quality-nomination-tag-select-background-color: #ffffff;
+  --quality-nomination-details-confirmation-bacground-color: rgba(0, 0, 0, 0.5);
```

nit:

```suggestion
  --quality-nomination-details-confirmation-background-color: rgba(0, 0, 0, 0.5);
```

igual en todos los demás lugares donde está este typo.

---

Oops, ya quedó.